firefox-ios/Client/Frontend/Browser/TopTabDisplayManager.swift

```diff
@@ -270,11 +270,6 @@ class TopTabDisplayManager: NSObject {
         }
 
         refreshStore(shouldAnimate: true)
-
-        let notificationObject = [Tab.privateModeKey: isPrivate]
```

Comment:
This is not needed anymore since the legacy homepage was the only one listening to this notification